deploy/helm/magda/charts/storage-api/templates/deployment.yaml

```diff
@@ -24,13 +24,35 @@ spec:
             "node",
             "/usr/src/app/component/dist/index.js",
             "--listenPort", "80",
-            "--bucket", "", # fill this in
-            "--minioAccessKey", "", # fill this in
-            "--minioSecretKey", "", # fill this in
             "--minioEnableSSL", "true",
-            "--minioServerHost", "", # fill this in
-            "--minioServerPort", ""  # fill this in
         ]
+        env:
+        - name: MINIO_HOST
```

Comment:
Realise I'm getting really annoying now, but only the SECRET_KEY and ACCESS_KEY should really be secrets, everything else belongs as a value that comes through from `values.yml`. Just makes it a bit easier to configure, because secrets are annoying (but necessary for keeping things secret)

Reply:
Fair point. Added values in the `deployment.yaml` file.